### ISSUE TITLE
Feature background disapear randomly on addPostFrameCallback

### ISSUE THREAD

**Issue by EliuTimana** [ayalma/feature_discovery#19]
when I have multiple features to show up at my ap startup the first one's background disappears randomly, even if I restart the app, it works perfectly when I activate it with a button.

![Imgur](https://i.imgur.com/83SXkeX.png)

As you can see the feature's background disappears, I put a dark background to show you that the description text is there.

This is the result when I trigger the sequence with a button, everything OK.

![](https://i.imgur.com/1vxITqw.png)

Here is my code.
```dart
//helper class
class FeatureManager {
  static const keyMenuSearch = 'key_menu_search';
  static const keyButtonSearch = 'key_button_Search';
  static const featuresSearch = {keyMenuSearch, keyButtonSearch};
}

@override
void initState() {
  SchedulerBinding.instance.addPostFrameCallback((Duration duration) {
    showFeatures();
  });
    super.initState();
}

void showFeatures() {
    FeatureDiscovery.clearPreferences(context, FeatureManager.featuresSearch).then((_) {
      FeatureDiscovery.discoverFeatures(context, FeatureManager.featuresSearch);
    });
}
```

**Comment by EliuTimana** [comment]
After several attempts I found that this fixes the problem, I don't know why, I'm not an expert, but for now, it solves the problem.

I also used a one-second custom duration and it worked, less than that won't work.

```dart
  @override
  void initState() {
    SchedulerBinding.instance.addPostFrameCallback((Duration duration) {
      Future.delayed(duration).then((_) {
        showFeatures();
      });
    });
    super.initState();
  }
```

**Comment by axel-op** [comment]
Could you please tell if the problem also occurs if you use version `0.7.0` of this package?

**Comment by EliuTimana** [comment]
> Could you please tell if the problem also occurs if you use version `0.7.0` of this package?

Nope, with `0.7.0` it works fine, but, what is the way to clear the preferences? I use the `clearPreferences()` functions to restart all the features, it doesn't exists in previous versions.

**Comment by kashlo** [comment]
Same problem
version: "0.13.0"

**Comment by guoz2013** [comment]
The latest version has the same problem
v0.14.0
